tests/smc/test_smc.py

```diff
@@ -133,6 +133,18 @@ def test_unobserved_categorical(self):
 
         assert np.all(np.median(trace["mu"], axis=0) == [1, 2])
 
+    def test_parallel_custom(self):
+        def _logp(value, mu):
+            return -((value - mu) ** 2)
+
+        def _random(mu, rng=None, size=None):
+            return rng.normal(loc=mu, scale=1, size=size)
+
+        with pm.Model():
+            mu = pm.CustomDist("mu", 0, logp=_logp, random=_random)
```

Comment:
This way we test both kinds of CustomDist with customized dispatched methods. You can use the `_dist` from my example

```suggestion
            mu = pm.CustomDist("mu", 0, dist=_dist, random=_random)
```